benchmark_runner/jupyterlab/templates/run_workload/run_workload.sh

```diff
@@ -0,0 +1,5 @@
+# Follow the stops:
+# git clone https://github.com/redhat-performance/benchmark-runner
+# update environment variables i.e. ('WORKLOAD','vdbench_vm') in benchmark_runner/main/environment_variables.py
+# run from benchmark-runner root path
+PYTHONPATH=. python3 benchmark_runner/main/main.py
```

Comment:
main.py accepts command line arguments (which is the normal practice for commands, rather than using environment variables), so the script to run it should too.

Reply:
Updated